latency-breakdown.md

```diff
@@ -105,26 +113,36 @@ read handle duration = read value duration =
     tidb_tikvclient_rpc_net_latency_seconds{store="?"}
 ```
 
-Get requests are sent directly to TiKV via a batched gRPC wrapper. The detail is in [batch client section](#batch-client).
+The `tidb_tikvclient_request_seconds{type="Get"}` records a duration of get requests which are sent directly to TiKV via a batched gRPC wrapper. For more details about the preceding batch client duration, such as `tidb_tikvclient_batch_wait_duration`, `tidb_tikvclient_batch_send_latency`, and `tidb_tikvclient_rpc_net_latency_seconds{store="?"}`, refer to the [Batch client](#batch-client) section.
+
+The `tikv_grpc_msg_duration_seconds{type="kv_get"}` duration is calculated as:
+
+<!--TODO: confirm the formula here-->
 
 ```text
 tikv_grpc_msg_duration_seconds{type="kv_get"} =
     tikv_storage_engine_async_request_duration_seconds{type="snapshot"} +
     tikv_engine_seek_micro_seconds{type="seek_average"} +
     read value duration +
-    read value duration
+    read value duration(from disk)
```

Comment:
`read value duration` stands for a general read from RocksDB, and it can read either from memory or disk. TiKV has short-value optimization, when the optimization is hit, one read action is enough, otherwise, two read actions are required. We may change it as:

```text
tikv_grpc_msg_duration_seconds{type="kv_get"} =
    tikv_storage_engine_async_request_duration_seconds{type="snapshot"} +
    tikv_engine_seek_micro_seconds{type="seek_average"} +
    read value duration +
    read value duration(non-short value)
```

Because the duration of reading from memory can be ignored, the doc only describes the latency of reading from disk in the following paragraph.